src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ParameterBinder.cs

```diff
@@ -39,17 +71,27 @@ public class ParameterBinder
         }
 
         /// <summary>
-        /// Initializes a new instance of <see cref="ParameterBinder"/>.
+        /// <para>This constructor is obsolete and will be removed in a future version. The recommended alternative
+        /// is the overload that also takes an <see cref="IModelValidatorProvider"/> and <see cref="ILoggerFactory"/>
+        /// instead of a <see cref="IObjectModelValidator"/>.</para>
```

Comment:
`instead of an ...` here and in the `[Obsolete(...)]` text. Same for the other obsolete constructor.